### PR TITLE
[AP] Cut AP release 2.10.0

### PR DESCRIPTION
**Pull request by lijamie98**

### CODE REVIEW

File: ap_versioned_docs/version-2.10.0/admin-guide/custody-services/configuration.mdx [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+
+</CodeExample>
+
+Start the Custody Server using Gradle or Docker.
+
+<CodeExample>
+
+```bash
+./gradlew service-runner:bootRun --args=--custody-server
+docker run stellar/anchor-platform:2.9.0 --custody-server
```
Comment (philipliu) [inline ap_versioned_docs/version-2.10.0/admin-guide/custody-services/configuration.mdx:64]: Should this be `2.10.0`? There are a few other places in this PR that reference 2.9.0.

**Review by philipliu — COMMENTED** [review]
Hm I don't see the new version in the deployment.

**Comment by philipliu** [comment]
Also, what is the difference between 2.9.0 and 2.10.0? Should we add docs for the new Kafka security settings?

File: ap_versioned_docs/version-2.10.0/admin-guide/custody-services/configuration.mdx (hunk 1 above)
Comment (lijamie98) [reply ap_versioned_docs/version-2.10.0/admin-guide/custody-services/configuration.mdx]: fixed

**Comment by lijamie98** [comment]
> Also, what is the difference between 2.9.0 and 2.10.0? Should we add docs for the new Kafka security settings?

The changes are Kafka security and bug fixes. They are only defined in YAML file.

**Comment by philipliu** [comment]
> The changes are Kafka security and bug fixes. They are only defined in YAML file.

Do we link the version-specific YAML anywhere in the docs? I think maybe we should otherwise, they would need to find the corresponding version in the AP repo.

**Comment by lijamie98** [comment]
> Also, what is the difference between 2.9.0 and 2.10.0? Should we add docs for the new Kafka security settings?

There is no difference. The new Kafka security settings are documented in the YAML file.

**Comment by philipliu** [comment]
> There is no difference. The new Kafka security settings are documented in the YAML file.

Do we expect the developers to know where the YAML file lives in our codebase? They would need to know how to find the version-specific configuration.

**Comment by lijamie98** [comment]
Let's discuss in the stand-up.

**Comment by JiahuiWho** [comment]
Is this gonna create version 2.10.0 in the dropdown menu and make it latest?

**Comment by lijamie98** [comment]
> Is this gonna create version 2.10.0 in the dropdown menu and make it latest?

It is suppose to. Since this is manual process, I will track it down if it does not.

**Review by JiahuiWho — APPROVED** [review]

**Pull request merged**